src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -45,28 +46,6 @@ static VisualStudioInstanceFactory()
         public VisualStudioInstanceFactory()
         {
             AppDomain.CurrentDomain.AssemblyResolve += AssemblyResolveHandler;
-            AppDomain.CurrentDomain.FirstChanceException += FirstChanceExceptionHandler;
-        }
-
-        private static void FirstChanceExceptionHandler(object sender, FirstChanceExceptionEventArgs eventArgs)
```

Comment:
I would argue that ship has sailed already.  Consuming these binaries on the project system side, its not really possible to avoid using xunit without duplicating a lot of code. I don't feel bad about telling people it only works with xUnit